### PR TITLE
[#4882] improvement(core): Add some ut for MySQL and PostgreSQL storage backend.

### PR DESCRIPTION
**Pull request by yuqi1129**
### What changes were proposed in this pull request?

Add some UTs to test MySQL and PostgreSQL storage backend.

### Why are the changes needed?

To improve code robustness.

Fix: #4882 

### Does this PR introduce _any_ user-facing change?

N/A.

### How was this patch tested?

N/A

### CODE REVIEW

**Review by jerryshao — APPROVED** [review]

**Comment by jerryshao** [comment]
@mchades can you please take another look?

File: core/src/test/java/org/apache/gravitino/storage/TestEntityStorage.java [hunk 1]
```diff
@@ -151,14 +183,16 @@ private void destroy(String type) {
...
+    } else if (type.equalsIgnoreCase("h2") || type.equalsIgnoreCase("mysql")) {
       dropAllTables();
       File dir = new File(DB_DIR);
       if (dir.exists()) {
         dir.delete();
       }
 
       FileUtils.deleteQuietly(new File(H2_FILE));
+    } else if (type.equalsIgnoreCase("postgresql")) {
+      // Do nothing
```
Comment (mchades) [inline core/src/test/java/org/apache/gravitino/storage/TestEntityStorage.java:195]: Why doesn't the PG backend need to clear existing resources like tables and files?

Comment (yuqi1129) [reply core/src/test/java/org/apache/gravitino/storage/TestEntityStorage.java:195]: PG has the concept database and schema, and each catalog uses a different database name. For PostgreSQL catalog:

The database in PosgreSQL is equivalent to the Gravitino catalog, and the schema is equivalent to the Gravitino  database.

**Comment by mchades** [comment]
Does this PR need to be backported to branch-0.6?

**Review by mchades — APPROVED** [review]

**Comment by jerryshao** [comment]
We don't have PG support in 0.6.

**Pull request merged**